pkg/kubelet/metrics/metrics.go

```diff
@@ -780,6 +783,17 @@ var (
 		},
 	)
 
+	// AdmissionRejectionsTotal tracks the number of failed admission times, currently, just record it for pod additions
+	AdmissionRejectionsTotal = metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Subsystem:      KubeletSubsystem,
+			Name:           AdmissionRejectionsTotalKey,
+			Help:           "Cumulative number pod admission rejections by the Kubelet.",
+			StabilityLevel: metrics.ALPHA,
+		},
+		[]string{"reason"},
```

Comment:
looks like `reason` is unbounded, so there maybe a possibility for the cardinality of this metric to explode.. wondering if we could use a more bounded label? cc @logicalhan

Reply:
Any suggestion here about what label to use?

Reply:
I apologize for the delay, but I don't have any ideal solutions other than completely removing the reason label if it's not essential. Alternatively, if there are specific reasons we need to track, we could just focus on populating those and disregarding the others.

Reply:
@lowang-bh why did you mark this as resolved? I don't think it's been addressed.

I'd suggest creating an allow-list of known reasons, and only recording reasons from that list, along with an "other" category.